dev/tasks/tasks.yml

```diff
@@ -157,6 +157,7 @@ groups:
     - example-*
     - wheel-*
     - python-sdist
+    - r-binary-packages
```

Comment:
Do we need this?
I think that the `nightly` group isn't maintained (but misused: ARROW-18094).

Reply:
I was initially using it for testing but we can drop it. I agree on ARROW-18094